theano/gpuarray/opt.py

```diff
@@ -1639,7 +1646,8 @@ def local_abstractconv_gemm(node):
             if prod1 > prod2:
                 rval = GpuCorrMM_gradWeights(border_mode,
                                              subsample,
-                                             filter_dilation)(
+                                             filter_dilation,
+                                             unshared)(
```

Comment:
Actually, `unshared` is redundant because the outer thing has a check for `not unshared`. I've removed it.